jekyll/_cci2/credits.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: classic-docs
+title: Using Credits
+categories: [how-to]
+description: How to leverage CircleCI credits
+---
+
+This document describes the basics of using credits with CircleCI. If you are on the legacy CircleCI container-based plan you may want to consider consulting the document on [using containers]({{site.baseurl}}/2.0/containers). If you want to switch from using containers to using credits, please [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) requesting so.
+
+## Overview
+
+The CircleCI credit-based usage plans enable you to only pay for what you use, while also providing flexibility in customizing and scaling your team's CI solution. Credits are used by the minute at varying rates according to what build configuration you use.
+
+The right plan for you and your team will depend on several factors:
+
+- How many users are on your team
+- How much support you need (community support, standard support, or premium support)
+- If you want access to different machine-types
+- If you need features such as Docker Layer Caching, concurrent builds, or build history
```

Comment:
For our performance plan, we don't differentiate anything yet for concurrency or build history, but may in the future. I don't see a problem with leaving these here, but FYI.

Reply:
Gotcha! Thanks!